docs/services/data/capture/_index.md

```diff
@@ -43,20 +61,30 @@ To capture data from one or more machines, you must first add the [data manageme
 2. Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
 3. Select the `data management` type, then either use the suggested name or specify a name for your data management service, for example `data-manager`.
 4. Click **Create**.
-5. On the panel that appears, you can manage the capturing and syncing functions and specify the **Directory**, the sync **Interval** and any **Tags** to apply to captured data.
-
-   If the sync **Interval** or the **Directory** is not specified, the data management service captures data at the default frequency every 0.1 minutes (after every 6 second interval) in the default `~/.viam/capture` directory.
+5. On the panel that appears, you can manage the capturing and syncing functions.
+   {{< tabs >}}
+   {{% tab name="RDK" %}}
+   Specify the **Directory**, the sync **Interval** and any **Tags** to apply to captured data.
 
-   {{< alert title="Info" color="info" >}}
-   If you change the directory for data capture only new data is stored in the new directory.
-   Existing data remains in the directory where it was stored.
-   {{< /alert >}}
+If the sync **Interval** or the **Directory** is not specified, the data management service captures data at the default frequency every 0.1 minutes (after every 6 second interval) in the default `~/.viam/capture` directory.
 
-6. Click the **Save** button in the top right corner of the page.
+{{< alert title="Info" color="info" >}}
+If you change the directory for data capture only new data is stored in the new directory.
+Existing data remains in the directory where it was stored.
+{{< /alert >}}
+{{% /tab %}}
+{{% tab name="micro-RDK" %}}
+Specify the sync **Interval**.
 
-![data capture configuration](/tutorials/data-management/data-management-conf.png)
+{{< alert title="Info" color="info" >}}
+With micro-RDK, the `capture_dir`, `tags`, and `additional_sync_paths` attributes are ignored and should not be configured.
```

Comment:
[optional] This is a minor existing bug that we could optionally fix in this PR.

These instructions are for the Builder UI, but this warning is for the raw JSON (and is repeated below in the raw JSON config section).

We could fix it by saying here 'With micro-RDK, the "Directory", "Tags", and "Additional paths" fields are ignored and should not be configured.'

Reply:
we don't currently differentiate between the builder and json mode. I have another ticket to review the data page, will fix then